chronicler/events/core/git.py

```diff
@@ -16,12 +16,15 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 
-from typing import Any
+import logging
+import re
 
-from cloudevents.http import CloudEvent
+from typing import Any, Generator
 
-from ...eventizer import Eventizer, uuid
+from cloudevents.http import CloudEvent
+from grimoirelab_toolkit.identities import generate_uuid
```

Comment:
In the module `eventizer` we have a function called `uuid`. Not sure if there are exactly the same. If they are, I think we should unify them.